charts/nuts-node/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "chart.fullname" . }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "chart.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "chart.selectorLabels" . | nindent 8 }}
+    spec:
+      volumes:
+        - name: nuts-config
+          configMap:
+            name: nuts-config
+        - name: nuts-ssl-secret
+          secret:
+            secretName: nuts-ssl-secret
+        - name: nuts-data-pv
+          persistentVolumeClaim:
+            claimName: nuts-data-pvc
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "chart.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: NUTS_CONFIGFILE
+              value: "/opt/nuts/nuts.yaml"
+          volumeMounts:
+            - name: nuts-config
+              mountPath: /opt/nuts/nuts.yaml
+              subPath: nuts.yaml
+              readOnly: true
+            - name: nuts-ssl-secret
+              mountPath: /opt/nuts/ssl
+              readOnly: true
+            - name: nuts-data-pv
+              mountPath: /opt/nuts/data
+          ports:
```

Comment:
all ports and probe settings use the default. When settings change, these will have to change as well. It's either a todo for the future or something to pickup now.

Reply:
Good point! I've actually tried modifying this before, but the config seems to contain address props, and no separate port numbers. 

Do you have a preferred solution? Do the config values *always* end with `:<port_number>`? Or can they also simply be hostnames? Assuming it should be based on the following two properties:
```
http.default.address                       :1323
network.grpcaddr                           :5555
```

Reply:
You will most certainly see IP's in the config because of being able to bind to specific interfaces (separate internal IPs from external ones). But I don't know if that is a situation for k8s. The pod will only have a single IP and the ingress will handle anything coming from outside? You could therefore make the choice to limit these settings to ports only. (and document it)?